localstack/services/events/events_listener.py

```diff
@@ -0,0 +1,67 @@
+import json
+import re
+import tempfile
+import uuid
+
+from requests.models import Response
+
+from localstack.constants import TEST_AWS_ACCOUNT_ID, MOTO_ACCOUNT_ID
+from localstack.services.generic_proxy import ProxyListener
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import to_str, save_file, TMP_FILES, mkdir
+
+EVENTS_TMP_DIR = tempfile.gettempdir() + '/cw_events/'
```

Comment:
Would be better if we could make this path relative to `config.TMP_FOLDER` (so the user can customize this):
```
import os
...
from localstack import config
...
EVENTS_TMP_DIR = os.path.join(config.TMP_FOLDER, 'cw_events')
```

Reply:
done